editor/components/icon-button/index.js

```diff
@@ -14,9 +14,8 @@ function IconButton( { icon, children, label, className, ...additionalProps } )
 	const classes = classnames( 'editor-icon-button', className );
 
 	return (
-		<Button { ...additionalProps } aria-label={ label } className={ classes }>
+		<Button { ...additionalProps } aria-label={ label } className={ classes } data-level={ children ? children : null }>
```

Comment:
Sorry, I merged early here. Regardless, would appreciate some help with fixing it. 💖